src/StimulusBundle/.gitignore

```diff
@@ -1,5 +1,6 @@
 .php-cs-fixer.cache
 .phpunit.result.cache
 composer.lock
+var/
```

Comment:
I'd added it because when running the testsuite it created a failure html-file in `var/browser/source` and to not accidentally add it git. I'll create a PR.